pkg/vault/contracts/VaultAdmin.sol

```diff
@@ -286,12 +287,15 @@ contract VaultAdmin is IVaultAdmin, VaultCommon, Authentication, VaultGuard {
         for (uint256 i = 0; i < poolTokens.length; ++i) {
             IERC20 token = poolTokens[i];
 
-            (totalSwapFees[i], totalYieldFees[i]) = _aggregateFeeAmounts[pool][token].fromPackedBalance();
+            // If a feeToken is specified, all reported fees will be zero except for that token.
```

Comment:
We should have a test for this change. I believe we're only testing the ProtocolFeeController.